src/assets/_project/_blocks/components/forms/qg-recaptcha.js

```diff
@@ -66,66 +64,49 @@ import keys from '../../data/qg-google-keys';
     },
 
     /**
-     * From SWE4 onwards footer feedback is AJAX based
+     * from swe4 onwards footer feedback is ajax based
+     * footerFeedbackRecaptcha function handles recaptcha on ajax based form.
+     * This function creates a submit event
+     * Call submission handler api
+     * Get and display success message
      * @return {undefined}
      **/
     footerFeedbackRecaptcha: function() {
```

Comment:
This is badly named looking at the comments.  can you renamed to 
footerFeedbackSubmitWithRecaptchaCheck

Reply:
updated